src/clj/rems/validate.clj

```diff
@@ -52,10 +52,13 @@
 
 (defn validate []
   (log/info "Validating configuration")
-  (when-some [oidc-attribute (:oidc-userid-attribute env)]
+  (when-let [old-attributes (some-> env :oidc-userid-attribute list flatten)]
```

Comment:
I don't think I have ever seen this use of `list flatten` :sweat_smile:. So it handles singular and sequential version.